### DISCUSSION TITLE
Business Privacy Policy

### DISCUSSION THREAD

**Discussion by MattiSG** [OpenTermsArchive/terms-types#20] [Added]
Same as Privacy Policy, but targeting Commercial Partners instead of End Users.

The triptych would thus be:

- writer: service provider
- audience: commercial partners
- object: commercial partners personal data

### [SkyScanner Corporate Business Contact Privacy Notice](https://www.partners.skyscanner.net/privacy-policy?hsLang=en)

> Skyscanner processes your personal data if you are employed by or associated with Skyscanner’s  corporate partners, suppliers, vendors or agencies in order to establish you as a point of contact for that organisation, onboard Your organisation to our platform (where relevant), communicate with you about the goods and services your organisation provides (or the services we may provide your organisation), and for internal administrative purposes. Your personal data is shared with our service providers, where necessary, some of which may be located outside the UK and the European Economic Area. This Corporate Business Contact Privacy Notice explains in detail how we process your personal data, and also includes information on your data protection rights, such as your right to object to certain processing activities including profiling.

**Comment by MattiSG** [comment]
### [Bolt Privacy Notice for Drivers](https://bolt.eu/en/legal/ie/privacy-for-drivers/)

**Comment by MattiSG** [comment]
I'm not sure about the name “Commercial Privacy Policy”. While an interesting symmetry to “Commercial Terms”, I find the title ambiguous: it could just as well be about the commercial usage of private data. I would thus prefer “Business Partners Privacy Policy”.

**Comment by MattiSG** [comment]
## Recap table of provided examples

| URL                                                       | Title                                     | Service    |
|-----------------------------------------------------------|-------------------------------------------|------------|
| [URL](https://www.partners.skyscanner.net/privacy-policy) | Corporate Business Contact Privacy Notice | Skyscanner |
| [URL](https://bolt.eu/en/legal/ie/privacy-for-drivers/)   | Privacy Notice for Drivers                | Bolt       |
| [URL](https://www.shopify.com/legal/privacy/merchants)    | Privacy for Merchants                     | Shopify    |

## Relevance

These documents describe how intermediation platforms handle the personal data of individuals acting as or on behalf of business users.

## Tryptich

- Writer: `intermediation service provider`
- Audience: `business user`
- Object: `personal data of business users and of people acting on their behalf`

## Naming

### Intent

Align name with other existing and upcoming document types. Thus:
- Prefer “Business” or “Business User” over “Commercial”, as an increasing part of our vocabulary relating to intermediation service providers is coming from the [P2B regulation](https://eur-lex.europa.eu/eli/reg/2019/1150/oj).
- Use “Privacy Policy” in a symmetry to the `Privacy Policy` document type targeting end users.

### Proposals

1. Business Privacy Policy
2. Business Users Privacy Policy
3. Commercial Privacy Policy

**Reply by martinratinaud** [reply]
In order to be in line with the audience, I would go for `Business Users Privacy Policy`

**Reply by clementbiron** [reply]
As there is no mention of  the audience in the naming of other documents types, i would prefer `Business Privacy Policy`

**Reply by lucielechardoy** [reply]
I would prefer 1. or 2. and probably more 2. Business users privacy policy (as business privacy policy could also sound like the privacy policy of the platform in general).
I'm checking to see if I can find other documents that could fit this new category.

**Reply by lucielechardoy** [reply]
Just found a third example, Shopify's Privacy for Merchants https://www.shopify.com/legal/privacy/merchants

**Reply by MattiSG** [reply]
As pointed out by @clementbiron, after going through the existing document types, we don't usually mention the audience explicitly. While we could question that, `Business Privacy Policy` seems more in line with the current naming habits.

**Comment by MattiSG** [comment]
Added in https://github.com/ambanum/OpenTermsArchive/pull/944!
